services/identity/src/main/java/com/crapi/controller/AuthController.java

```diff
@@ -177,4 +179,24 @@ public ResponseEntity<JwtResponse> loginWithTokenV2(
     }
     return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(jwt);
   }
+
+  /** @return success or failure of password updation. */
+  @PostMapping("/reset")
```

Comment:
I would suggest something like `/reset-test-users`, `/reset` is more like password reset for any user right?

Reply:
done